network-api/networkapi/templates/pages/buyersguide/category_page.html

```diff
@@ -22,7 +22,13 @@ <h2>({{ category.name }}) Primary related articles (First 3 of list of 6)</h2>
           <div class="tw-m-4">
             {% include "fragments/buyersguide/related_reading.html" with articles=primary_related_articles all_articles="True" %}
           </div>
-          
+
+            {% with cta=featured_cta %}
+              {% if cta %}
+                {% include "fragments/buyersguide/call_to_action_box.html" with icon=cta.sticker_image heading=cta.title body=cta.content link_text=cta.link_label link_href=cta.get_target_url %}
+              {% endif %}
+            {% endwith %}
```

Comment:
**Question** Why is this indented a level deeper than its sibling above?

Reply:
Typo, that is now updated :)